app/core/store/modules/seasonalLeague.js

```diff
@@ -356,6 +365,12 @@ export default {
       await topCodePointsRankingPromise
 
       commit('setCodePointsRanking', { leagueId: leagueId, ranking: codePointsRankingInfo })
+
+      getCodePointsPlayerCount(leagueId)
+          .then((playerCount) => {
+            commit('setCodePointsPlayerCount', Number(playerCount));
```

Comment:
Not blocking but curious if there is a difference between `Number` and `parseInt` for conversion of string to number.

Reply:
I just searched for it. There is a difference
```
// parsing:
parseInt("20px");       // 20
parseInt("10100", 2);   // 20
parseInt("2e1");        // 2

// type conversion
Number("20px");       // NaN
Number("2e1");        // 20, exponential notation
```